rqt_gui_cpp/CMakeLists.txt

```diff
@@ -23,22 +40,34 @@ set(rqt_gui_cpp_HDRS
 
 qt5_wrap_cpp(rqt_gui_cpp_MOCS ${rqt_gui_cpp_HDRS})
 
-add_library(${PROJECT_NAME} ${rqt_gui_cpp_SRCS} ${rqt_gui_cpp_MOCS})
-target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES} Qt5::Widgets)
+ament_export_dependencies(Qt5Widgets)
 
 if(APPLE)
+  # TODO(mlautman): validate on mac
   set_target_properties(${PROJECT_NAME} PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
 endif()
 
-install(FILES plugin.xml
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+ament_export_include_directories(include)
+
+add_library(${PROJECT_NAME}
+  ${rqt_gui_cpp_SRCS}
+  ${rqt_gui_cpp_MOCS})
+
+target_link_libraries(
+  ${PROJECT_NAME}
+  ${qt_gui_cpp_LIBRARIES}
+  ${ament_LIBRARIES}
```

Comment:
This isn't a valid variable afaik.

Reply:
I got this from https://github.com/ros-visualization/qt_gui_core/blob/0c2aaaaa4340f2a810abd91f6f328236c6beb1f9/qt_gui_cpp/src/qt_gui_cpp_shiboken/CMakeLists.txt#L58

Reply:
${ament_LIBRARIES} is my fault. It's not a valid variable